### ISSUE TITLE
[AIR 51] Incorrect text selection when select text to left/right side of the text field

### ISSUE THREAD

**Issue by ylazy** [airsdk/Adobe-Runtime-Support#3311]
Please see the clip below for more information (the issue does not happen in Air 32, happen in Air 51)

https://github.com/airsdk/Adobe-Runtime-Support/assets/6010183/dfb66f0d-788a-488d-97c3-7d969d71e469

When I clicked on the middle of the line (hold the mouse) then moved to the left/right. When the mouse fall outside of the text field, the selection was incorrect and blink.

In the video I've tested with the left side only. In fact the issue happens with the right side too.

Thanks!

**Comment by ylazy** [comment]
I added the code below to easily reproduce the problem:

```as
FontManager.embed(Arial); // embed the Arial font

var format:TextFormat = new TextFormat();
format.font = "Arial";

var tf:TextField = new TextField();
tf.defaultTextFormat = format;
tf.border = true;
tf.width = 300;
tf.height = 200;
tf.wordWrap = false;
tf.multiline = true;
tf.type = TextFieldType.DYNAMIC;
tf.selectable = true;
tf.embedFonts = true;
tf.text = "first line\nsecond line\nthird line";
addChild(tf);
```

The issue only happens with **embed fonts**.

**Comment by ylazy** [comment]
Fixed on AIR SDK 51.0.1.4.

Thanks!
